model/src/channel/permission_overwrite.rs

```diff
@@ -42,7 +42,7 @@ pub struct PermissionOverwrite {
     pub kind: PermissionOverwriteType,
 }
 
-#[derive(Clone, Debug, Eq, Hash, PartialEq)]
+#[derive(Clone, Copy, Debug, Eq, Hash, PartialEq)]
```

Comment:
This enum seems to be missing static assertion tests. Good time to add them since you're adding a trait to it?

Reply:
Done.